frontend/packages/dev-console/integration-tests/cypress.json

```diff
@@ -23,10 +23,10 @@
   "screenshotsFolder": "../../../gui_test_screenshots/cypress/screenshots",
   "videosFolder": "../../../gui_test_screenshots/cypress/videos",
   "env": {
-    "TAGS": "@smoke and not @manual"
+    "TAGS": "@smoke or @regression and not @manual"
   },
   "retries": {
     "runMode": 1,
     "openMode": 0
   }
-}
+}
```

Comment:
Add the empty line at end of every file

Reply:
This is because of the automatic formatting whenever i save the file. Automatic formatting is caused by the extension that i have used in vs code - "cypress-cucumber-steps"